plutus-contract/src/Plutus/Contract/CardanoAPITemp.hs

```diff
@@ -0,0 +1,161 @@
+{-# LANGUAGE BlockArguments           #-}
+{-# LANGUAGE DisambiguateRecordFields #-}
+{-# LANGUAGE EmptyCase                #-}
+{-# LANGUAGE FlexibleContexts         #-}
+{-# LANGUAGE GADTs                    #-}
+{-# LANGUAGE NamedFieldPuns           #-}
+{-# LANGUAGE TypeApplications         #-}
+-- Code temporarily copied over from cardano-api.
```

Comment:
why? when will we get rid of it? better out of your head!

Reply:
Well, it's not in my head but in Jann's 😅, but I'll try to add a clarification.